Runtime/Process/EventProcess.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Zinnia.Process
+{
+    using Malimbe.XmlDocumentationAttribute;
+    using UnityEngine;
+    using UnityEngine.Events;
+
+    /// <summary>
+    /// Emit <see cref="UnityEvent"/> when processed.
```

Comment:
```suggestion
    /// Raises a <see cref="UnityEvent"/> when processed.
```

"Raising an event" is proper C# terminology. I know the existing codebase talks about "emitting" most of the time, but we should start getting this right in new changes at least.